### PR TITLE
[ready] NSV Jeppison

### PR DESCRIPTION
**Pull request by Vasily2013**
## About The Pull Request

Incoming message from Nanotrasen Fleet Command. A new ship classification has been prototyped and is ready for testing. This ship focuses heavily on ballistic, non-guided munitions and a minimal fighter compliment. 

## Why It's Good For The Game

Vago needs a partner

## Changelog
:cl:
add: A whole new map!
Tweak: Adds some hull plating to FOB! 
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
Add an anterchamber with some plasteel(to fix), radiation closet and some fluff stuff
изображение
Eh?
изображение
.
Where the unfiltered gas(not plasma, O2, CO2, N2, N2O) goes?
.
Either remove the water cooler or add some plastic cups on the table near it
изображение
INJECTOR, NOT VENT
изображение
Maybe put those crates into warehouse?
изображение
This place lacks either blood or synthetic limbs crate
изображение
The fuck is this floor? Also, put a cryo beaker near cryo chamber since not everyone got access to morgue
изображение
.
I think chemistry lacks pill bottles
.
Sparks?
изображение
Probably too much loot for 1 room? Gibs is a bit too much for maintenance either
изображение
I hope that's pink glitter, not plasma
изображение
Can't reach the tank storage unit
изображение
I cry as an Atmosian, getting jealous with that amount of canisters scientists have
изображение
Add showers and xenobio lockers to xenobio passage
изображение
Nice light
изображение
Actually this entire corridor of space looks obsolete. Mind replacing it with maint tunnel?
изображение
***Three*** techfabs
изображение
Not funny, didn't laugh
изображение
Bartenders, chemists, virologists will kill you for this ORM placement. What are roller beds for?
изображение
Make materials more organized here, please
изображение
...why?
изображение
D E C A L S
изображение
Imagine falling down right into medbay
изображение
.
Add railings to the stairs
.
I think this place lacks cables
изображение
Add tiles
изображение
Replace glass table with reinforced table
изображение
Prisoners ***will*** break into warden's office. The fuck is mass driver for? So unconfirmed changelings can escape?
изображение
.
4+4 lockers. 8 sec officers, you seriously?
.
ez sec loot
изображение
SHUTTERS
изображение
You lack a glass here
изображение
Telecomms **will** overheat
изображение
Wiring?
изображение
Why there are handcuffs?
изображение
***T I L E S***
изображение
overpriced
изображение
Merge this into one long tunnel or fill both rooms with some shit
изображение
>Enforcing MRP
>Putting a fucking nuclear warhead in MaA office
изображение
.
All hallways look very empty. Add decals, signs, vendors
.
What are torpedoes for? Add torpedo tubes
...
I do actually like this map

**Comment by Vasily2013** [comment]
Alright, adressing your critiques. 

Good ideas I like
-A larger airlock to grav gen could be fun, would also give a place for a SMES
-change toxins room to injector (This was a toxins coppied from another map)
-I'll move some crates into the warehouse.
-Lockers and showers to xenobio room
-remove light in toxins and direct it against the wall.
-find better place for ORM placement, also design anti-theft system
-decals
-signs
-add pets
-fully secure off the AI law boards.
-Add tiles to chapel (Look at commit history. I literally did this last)
-add a locker or two above MAA's office
-Torps were there before DI2, Will change to missiles later to help the fighter compliment.

Rebuttles to points I don't.
-Your rendering program isn't rendering wires
-not sure what "eh" means. That room is specifically for the plasma constrictors.
-Unfiltered gas goes outside. This can be redirected elsewhere as a fun in-round project.
-I'm about 90 percent sure that water coolers start with a bunch of paper cups.
-That crate is blood, look in the medical supply/storage for limbs
-Those sparks are a gib spawner
-Room copied off Box
-It's neither!
-You can move the firefoam or water tank depending on what you need.
-atmos can always ask to borrow some. In general though this is because I have no clue as to what toxins actually needs, so "enough" is my standard.
-that "corridor" is specifically to make it so cargo doesn't share a wall with toxins.
-Actually that's techfab, protolathe, protolathe
-that is actually glitter, and I think it's entertaining. I could change it to plasma.
-Roller beds are for dead miners.
-You think a NT restocking crew gives a damn about organization?
-Prevents high security armory from having shared wall with a public hallway.
-You actually fall into the Maint or the MAA office.
-Stairs have railings
-Janitors can add tiles
-If they try to break into the warden's office the warden shouldn't be too far away... as well the warden should secure his equipment better if they get it.
-I had extra space in sec. Got suggestions for one of those storage rooms?
-Lockers are anchored down, and secure your lockers. Those will also probably be the first pillaged by security officers.
-HOS has shutters. Button is over disposals.
-Tcomms overheating isn't a thing in this codebase.
-Handcuffs are obviously for throwing degenerates off of the bridge.
-"Overpriced?" I say Command office.
-The nuke has a name, its name is Antonio.

**Comment by ta3370** [comment]
If that ORM placement is a complaint, you're gonna HATE concordat xD. What do people have against _asking and communicating with others_ ? Inspiring communication should always be a goal, plus having wall-ORM *ALWAYS* leads to spergs using it to break into cargo, even further reducing communication.

**Comment by Firefox13** [comment]
> -not sure what "eh" means. That room is specifically for the plasma constrictors.

изображение
I actually meant this scrubber, but it looks like all scrubbers doubled. Stupid map diff render.
> -I'm about 90 percent sure that water coolers start with a bunch of paper cups.

Strangely, they do. Got really used to a box of paper cups near water cooler
изображение
> -I had extra space in sec. Got suggestions for one of those storage rooms?

A small office area.
> -Lockers are anchored down, and secure your lockers. Those will also probably be the first pillaged by security officers.

There is no delay in unwrenching the closet. Don't really understand the idea of security closets behind a single door. 
> -If they try to break into the warden's office the warden shouldn't be too far away... as well the warden should secure his equipment better if they get it.

Isn't this map is meant for low pop? Big chance we don't get a warden.

> -Handcuffs are obviously for throwing degenerates off of the bridge.

Put a box of handcuffs, then.
.
I'm fine with rest of the answers.
.
No blood bags?
изображение

**Comment by Vasily2013** [comment]
-Theoretically the bridge shouldn't need more than one pair of handcuffs at a time, and they should recover equipment from security if they use one. Two should be enough.
-added some blood to brigbay
-added some perma shutters

**Comment by Bokkiewokkie** [comment]
Why does the service sector have a total of three lathes all located within 30 tiles of eachother? The cook and the bartender can just walk to the service hall wich is literally at the end of the hallway.

**Comment by Vasily2013** [comment]
> Please talk to KMC about the way the overmap is handled now. I believe we do not use the overmap.dmm as a direct thing for each map anymore.

Map is on hold until starmap is merged specifically due to the fact of everything is changing.

**Comment by Vasily2013** [comment]
Please place all other issues and commentary here. 
https://github.com/BeeStation/NSV13/issues/608

File: _maps/map_files/Jeppison/job_changes.dm [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/datum/job/pilot/New()
+	..()
+	MAP_JOB_CHECK
+	total_positions = 2
+	spawn_positions = 2
+
+/datum/job/deck-tech/new()
```
Comment (francinum) [inline _maps/map_files/Jeppison/job_changes.dm:7]: Improper capitalization
```suggestion
/datum/job/deck_tech/New()
```

File: _maps/map_files/Jeppison/job_changes.dm [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/datum/job/pilot/New()
```
Comment (francinum) [inline _maps/map_files/Jeppison/job_changes.dm:1]:
```suggestion
/datum/job/fighter_pilot/New()
```

**Comment by Firefox13** [comment]
> Keep your chain of command filled.

like people ever care for it

I think it requires a light to see your bomb
изображение

I also found a telecrystal in cargo. It's either very cringe random spawn or you put it there. Really not a good idea!

wait wait is that random maint spawns? or circuits? please make it circuits!
изображение

"""airlocks""" lack scrubbers and vents
изображение

...can those rooms get increased to 3x3?
изображение

Add some maints there or transform it into bar backroom
изображение

can we get stabilized ration packs from Enterprise?
изображение

can you please organize it, i.e. it's own tile for every medkit type. also lacks oxygen medkits
изображение

add maint entry and those fucking things I forgot their names to windows
изображение
these things, for "privacy", so you don't get distracted ~~while you amputate cap's head~~
изображение

.
I think robotics lacks plasteel.
.

**Comment by Vasily2013** [comment]
- I'm not moving perma.
- [ ] Light added to the bomb testing area
- that TC would be from a random maint spawner. Not sure if it was real or not.
- Those are board spawners for tech storage.
- [x] Scrubbers and vents added
- No
- The reason their is no maint there is because Virology, a rather sensitive area, is directly below it.
- [x] O2 deprivation kits added, sorting is up to player discretion due to there being 5 types of medkits and only 4 cells.
- [x] Added curtains to surgery windows
- If you show me a decent way to put a maint airlock in there feel free to show me, because I don't see one that's pretty, and otherwise you have to run through very little space to get to a maint airlock.
- [x] Added a small amount of round start plasteel to robotics positions.

**Pull request merged**